File: 21C037/day6.py
t=int(input())
for i in range(t):
    a,b,x=[int(x) for x in input().split()]
    max=0
    for p in range(a,b+1):
        ans=(p//x)+(p%x)
        if(ans>=max):
            max=ans
    print(max)

File: 21C089/day6.java

import java.util.*;

public class day6 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();

        for (int i = 0; i < t; i++) {
            int a = input.nextInt();
            int b = input.nextInt();
            int x = input.nextInt();
            int max = 0;

            for (int j = a; j <= b; j++) {
                int as = (j / x) + (j % x);
                if (as >= max) {
                    max = as;
                }
            }

            System.out.println(max);
        }
    }
}